src/sourcify-chains.ts

```diff
@@ -790,6 +790,14 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
       "https://explorer.plexfinance.us/" + BLOCKSCOUT_SUFFIX,
     txRegex: getBlockscoutRegex(),
   },
+  "2000": {
+    // DogeChain Mainnet
+    supported: true,
+    monitored: true,
```

Comment:
Dogechain must not be monitored

```suggestion
    monitored: false,
```